railties/test/generators/actions_test.rb

```diff
@@ -31,13 +31,13 @@ def test_invoke_other_generator_with_full_namespace
   end
 
   def test_create_file_should_write_data_to_file_path
-    action :create_file, "lib/test_file.rb", "heres test data"
-    assert_file "lib/test_file.rb", "heres test data"
+    action :create_file, "lib/test_file.rb", "here's test data"
```

Comment:
`here's` is a contraction of `here is`

```suggestion
    action :create_file, "lib/test_file.rb", "here is test data"
```